pydrad/configure/configure.py

```diff
@@ -126,14 +127,16 @@ def setup_initial_conditions(self, root_dir, execute=True):
             with open(os.path.join(root_dir, filename), 'w') as f:
                 f.write(filestring)
         # NOTE: make sure we have needed permissions to run compile script
+		# Only do this on Unix-based systems
+        if not self.on_windows:
+            run_shell_command(
+                ['chmod', 'u+x', 'build_initial_conditions.bat'],
+                os.path.join(root_dir, 'Initial_Conditions/build_scripts'),
+                shell=False
+            )
         run_shell_command(
-            ['chmod', 'u+x', 'build_initial_conditions.bat'],
-            os.path.join(root_dir, 'Initial_Conditions/build_scripts'),
-            shell=False
-        )
-        run_shell_command(
-            ['./build_initial_conditions.bat'],
-            os.path.join(root_dir, 'Initial_Conditions/build_scripts')
+			['./build_initial_conditions.bat'],
+			os.path.join(root_dir, 'Initial_Conditions/build_scripts')
```

Comment:
I think the indentation got messed up here.
```suggestion
		['./build_initial_conditions.bat'],
		os.path.join(root_dir, 'Initial_Conditions/build_scripts')
```